core/src/main/scala/org/apache/spark/scheduler/ReplayListenerBus.scala

```diff
@@ -118,10 +118,12 @@ private[spark] class ReplayListenerBus extends SparkListenerBus with Logging {
       case e: HaltReplayException =>
         // Just stop replay.
       case _: EOFException if maybeTruncated =>
-      case _: IOException if maybeTruncated =>
-        logWarning(s"Failed to read Spark event log: $sourceName")
       case ioe: IOException =>
-        throw ioe
+        if (maybeTruncated) {
```

Comment:
I think this was already the behavior? if it doesn't match the 'if' it would just throw anyway

Reply:
Yes. I think, I simplified it in one block.

Reply:
Oh I see. Actually, what about just removing the second case? it's simpler to just let it throw.

Reply:
Thanks. updated